docs/geedocs/5.3.0/answer/176738.html

```diff
@@ -255,10 +255,10 @@ <h2 id="cache_configuration">Cache configuration</h2>
 (assuming your asset root is <code>/gevol/assets</code>; if not, modify the path accordingly).
 The asset cache size is controlled by <code>AssetCacheSize</code> and the asset version cache size is controlled by <code>VersionCacheSize</code>.
 Note that the asset version cache is generally larger than the asset cache because it may contain multiple versions of the same asset.</p>
-
-Also included within <code>misc.xml</code> is an experimental option, which allows for the user to disable cache purging while projects build. This is controlled by <code>DisablePacketLevelVersionCachePurge</code> and its default value is <code>false</code>.
 <br><br>
-In some instances, memory utilization may become problematic. An experimental feature is included which will allow for modifying aspects of the the XML cache. This is controlled through <code>/etc/opt/google/XMLparams</code>. It takes the form of a standard configuration file, and it is in the following format.
+Within <code>misc.xml</code>, the following options are deprecated, as enabled permanently: <code>DisablePacketLevelVersionCachePurge</code> and <code>ConsolidateListenerNotifications</code>.
```

Comment:
Minor: might change "as enabled permanently" to "as if they were enabled permanently".